app/models/work.rb

```diff
@@ -0,0 +1,30 @@
+class Work < ApplicationRecord
+  validates :category, presence: true
+  validates :title, presence: true, uniqueness: true
+  has_many :votes, dependent: :destroy
+
+  def self.top_ten(type)
+    array_by_type = self.where(category: type)
```

Comment:
I think these variables names could be better. If it were me, I would rename `array_by_type` to simply `works` or `categorized_works`, and `array_by_votes` to `sorted_works`